source/s3_request_messages.c

```diff
@@ -352,17 +353,18 @@ struct aws_http_message *aws_s3_create_multipart_upload_message_new(
     return NULL;
 }
 
-/* Create a new put object request from an existing put object request.  Currently just optionally adds part information
- * for a multipart upload. */
+/**
+ * Create a new put object request from an existing put object request.  Currently just optionally adds part information
+ * for a multipart upload.
+ **/
 struct aws_http_message *aws_s3_upload_part_message_new(
     struct aws_allocator *allocator,
     struct aws_http_message *base_message,
     struct aws_byte_buf *buffer,
     uint32_t part_number,
     const struct aws_string *upload_id,
     bool should_compute_content_md5,
```

Comment:
not for now, but we should really unify md5 with the rest of checksum at some point